nphysics_testbed2d/src/testbed.rs

```diff
@@ -50,11 +52,11 @@ fn usage(exe_name: &str) {
 }
 
 pub struct Testbed {
-    world: World<f32>,
+    world: Arc<Mutex<World<f32>>>,
```

Comment:
Is there a reason why you used a `Mutex` instead of a `RwLock`? The latter should be more flexible by allowing multiple simultaneous read-only locks.